### ISSUE TITLE
source highlighter rouge is not working with asciidoctorj 2.0.0

### ISSUE THREAD

**Issue by eiswind** [asciidoctor/asciidoctor-reveal.js#261]
```
<sourceHighlighter>rouge</sourceHighlighter>
<attributes>
<rouge-css>style</rouge-css>
```                                  
I tried with asciidoctor-revealjs 2.0.0 and revealjs 3.5.0 as suggested in the asciidoctor maven examples.

It renders as 
```
<pre class="highlight listingblock"><code class="java language-java">
```

I expected rouge support to be working with revealjs.

**Comment by bentolor** [comment]
This is plainly, because reveal.js itselfs only bundles `monokai.css` and `zenburn.css`.

You need to add new themews into `reveal.js/lib/css/`

**Comment by Mogztter** [comment]
I think we should support Rouge out-of-box. The syntax highlighting API changed in Asciidoctor 2.0.0 and we need to update the template accordingly:

https://github.com/asciidoctor/asciidoctor/blob/f4eaf65195e19caf99cabf386bd52fd2e2307977/lib/asciidoctor/converter/html5.rb#L155-L157

https://github.com/asciidoctor/asciidoctor/blob/f4eaf65195e19caf99cabf386bd52fd2e2307977/lib/asciidoctor/converter/html5.rb#L659-L668

https://github.com/asciidoctor/asciidoctor/blob/f4eaf65195e19caf99cabf386bd52fd2e2307977/lib/asciidoctor/converter/html5.rb#L253-L262

**Comment by obilodeau** [comment]
Fixed by merging #287. Will be part of the next release.
